db/migrate/20190313142540_add_shipping_category_to_shipping_methods_and_products.rb

```diff
@@ -0,0 +1,13 @@
+class AddShippingCategoryToShippingMethodsAndProducts < ActiveRecord::Migration
+  def up
+    # This is different from the equivalent Spree migration
+    #   Here we are creating the default shipping category even if there are already shipping categories
+    default_category = Spree::ShippingCategory.create!(name: "Default")
```

Comment:
Is it safe to assume that we don't have a shipping category yet? I guess so.

Reply:
Even if we have other shipping categories (we will in some production environments), I think it's good that we get this default one. We can then use it to make shipping category optional at some point...

Reply:
Okay, let me rephrase the question: Is it safe to assume that we don't have a `Default` shipping category yet? I guess this would fail then.

Reply:
There is neither an ActiveModel validation nor a DB constraint so we will have duplicates at most. Even in that case, I think we could just merge them later on right?

Reply:
ah, I see you question now Maikel.
Default duplicates will not be a problem, all we need here is a shipping category to assign to all products and methods without a shipping category. after this, the users will have to set shipping categories on products and methods as they create them. and admins will be able to manage shipping categories as they want (including this Default one).

Reply:
I agree with Pau, this will never fail as there is no constraint on unique shipping_category.name. And if we end up with duplicate names (there's no "Default" shipping category in any of our main instances right now), admins will be able to manage them as they want.